src/man/sssd-ad.5.xml

```diff
@@ -1356,6 +1370,56 @@ ad_gpo_map_deny = +my_pam_service
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>dyndns_dot_cacert (string)</term>
+                    <listitem>
+                        <para>
+                            This option specifies the file of the certificate
+                            authorities certificates (in PEM format) in order
```

Comment:
...path to the CA certificates bundle (in PEM format) used to verify...?

"certificate authorities certificates" is difficult to read, I think CA is more common.

Reply:
This text is actually a copy from nsupdate man page. I would prefer to keep it the same since it is actually nsupdate option.

Reply:
Ok, thank you.